### DISCUSSION TITLE
Apple-TV homebridge-ring

### DISCUSSION THREAD

**Discussion by cleversmart** [dgreif/ring#976] [Q&A]
For a month now the apple tv integration does not work. everything else, home on iphone and ipad works flawlessly. Cameras show as inactive on apple tv and also don't report when there is motion or doorbell activity. I'm using the latest version of homebridge-ring (11.0.5) and AppleTV 15.5.1. any ideas about this?

**Comment by m3rg3d0wn** [comment]
Same. Cameras via AppleTV no longer work. I don’t even see the settings in the Plug-in anymore.

**Comment by dgreif** [comment]
If the steam works on other devices, but not on AppleTV, then there is likely a bug in HomeKit.  Unfortunately there is nothing I can do to fix bugs on that end, so you will likely need to wait for the next tvOS update.

**Reply by m3rg3d0wn** [reply]
Did the settings to allow them to show via AppleTV go away?

I ended up switching to the Scypted plugin for the camera views.

**Reply by dgreif** [reply]
Ahh yes, I did remove the motion -> appleTV settings because the latest version of tvOS has built-in support for this feature

**Comment by bwtjr** [comment]
I have the same issue on my living room Apple TV, however, the Apple TV in my bedroom is able to access all 3 of my cameras without issue. I've tried restarting Homebridge, Apple TV, network, removing/reinstalling the plugin, signing out of iCloud. I am tempted to move to Scripted which several users have suggested, but it seems like it might be a little above my head.

**Comment by HOMESEERDUDE** [comment]
I have the reverse problem.  My new ring camera works on the apple tv but not in the homekit app.  Any suggestions?

**Reply by dgreif** [reply]
Reboot all your devices (including router). https://github.com/dgreif/ring/wiki/Camera-Troubleshooting
